drivers/net/can/sja1000/Kconfig

```diff
@@ -102,6 +102,12 @@ config CAN_SJA1000_PLATFORM
 	  on embedded systems with OpenFirmware bindings, e.g. if you
 	  have a PowerPC based system you may want to enable this option.
 
+config CAN_TS7800V2
+	tristate "TS-7800v2 FPGA CAN Controller"
```

Comment:
`tristate "TS-7800v2 FPGA CAN Controller"`
should be
`tristate "TS-7800-V2 FPGA CAN Controller"` to match produce numer